packages/flutter/lib/src/cupertino/text_form_field_row.dart

```diff
@@ -234,6 +235,7 @@ class CupertinoTextFormFieldRow extends FormField<String> {
                   placeholder: placeholder,
                   placeholderStyle: placeholderStyle,
                   contextMenuBuilder: contextMenuBuilder,
+                  spellCheckConfiguration: spellCheckConfiguration
```

Comment:
I _would_ do a trailing comma nitpick, but the autoformatter pass on the 19th will take care of it!